lib/src/constants.dart

```diff
@@ -15,7 +15,7 @@ const tag = 'binaries-v0.0.4';
 const cronetBinaryUrl =
     'https://github.com/google/cronet.dart/releases/download/$tag/';
 const cronetVersion = "86.0.4240.198";
-const wrapperVersion = "1";
+const wrapperVersion = "2";
```

Comment:
`const tag = 'binaries-v0.0.4';` needs to be updated as well? Or not because we're always compiling the wrapper for the client?

Reply:
Yes. Wrapper is always compiled on the client side. No need to update the tag.